server/src/attachments/attahcment.controller.ts

```diff
@@ -0,0 +1,28 @@
+import { Controller, Get, Param, Res, StreamableFile } from '@nestjs/common';
+import { Response } from 'express';
+import { Readable } from 'stream';
+import { Attachment } from './attachment.entity';
+import { AttachmentService } from './attachment.service';
+
+@Controller('attachments')
+export class AttachmentController {
+  constructor(private attachmentService: AttachmentService) {}
+
+  @Get('/:id')
+  async getAttachment(
+    @Res({ passthrough: true }) res: Response,
+    @Param('attachmentId') attachmentId: string
+  ) {
+    const attachment: Attachment = await this.attachmentService.getAttachment(attachmentId);
+    if (attachment) {
```

Comment:
If an attachment is not available, then 404 exception should be thrown!

Reply:
Makes sense, added!